lib/Sources/App/AppModel.swift

```diff
@@ -46,7 +46,12 @@ public class AppModel {
         Task {
             await tabsModel.performInitialLoad()
             if tabsModel.isEmpty {
-                self.activePageModel = buildPageModel(tabIndex: 1)
+                var urlFromClipboard: URL?
+                if UIPasteboard.general.hasURLs {
+                    urlFromClipboard = UIPasteboard.general.url
+                }
+
+                self.activePageModel = buildPageModel(tabIndex: 1, initialUrl: urlFromClipboard)
             }
         }
     }
```

Comment:
I'm wondering if this should be expanded to also do this when the empty page screen is already open? I had the app open already (on the empty page view with the search bar) and had to kill and restart it in order for the auto-paste to be triggered.

Reply:
A second issue is that if you say "yes" to the allow prompt, subsequent launches of the app (if it has zero tabs, and is killed) will silently paste-n-go to the url in the paste buffer without prompting which is pretty unexpected.

Reply:
So, the whole point of this is for smoother onboarding when navigating to a BLE enabled web app. Your need to kill and relaunch the app wouldn't have happened to a user who was prompted to download Topaz  and is going through the initial onboarding process.

As for the second issue, that's true. Didn't think about that. I can easily make it more specific to the app's first launch with an app storage property.

Thoughts?

Reply:
Convo from you, me, and Jon: the first time the app notices a user has a URL in their clipboard, do this prompt. After that, don't do this behavior.